src/controllers/contacts.ts

```diff
@@ -650,7 +683,7 @@ export const getLatestContacts = async (
   try {
     const dateToReturn = decodeURI(req.query.date as string)
     const local = moment.utc(dateToReturn).local().toDate()
-    const where: { [k: string]: any } = {
+    const where: { tenant: number; updatedAt: { [Op.gte]: Date } } = {
```

Comment:
nvm I think there might be too many differences between the where clauses but it seems like a good amount of them use `tenant`

Reply:
Literally basically every single query should have `tenant` in it, because relay can also run in multitenant mode where there are many "owner" contacts